### DISCUSSION TITLE
Error While Creating A Secret<title>

### DISCUSSION THREAD

**Discussion by TTsGithubrep** [community/community#113375] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Unfortunately, I am not able to add secret to github, it gives me an error. Anybody can help me? 

Failed to add secret. Secret names can only contain alphanumeric characters ([a-z], [A-Z], [0-9]) or underscores (_). Spaces are not allowed. Must start with a letter ([a-z], [A-Z]) or underscores (_).

Note: secret is appropriate to the rule, actually.

**Comment by lin20260407** [comment]
Bro, maybe you copied the wrong password to replace.
For example, I've used SECRET in my repository before, and the SECRET name is ACCESS_TOKEN. The secret to be protected is just COPY TOKEN.

Hope this helps, have a nice day!
